src/utils/clientHelpers.ts

```diff
@@ -21,22 +21,22 @@ export function getInitializationOptions() {
     here is to make room for this to be added to a configuration builder when
     we tackle #791
   */
-  const rootModulePaths = config('terraform-ls').get<string[]>('rootModules', []);
-  const terraformExecPath = config('terraform-ls').get<string>('terraformExecPath', '');
-  const terraformExecTimeout = config('terraform-ls').get<string>('terraformExecTimeout', '');
-  const terraformLogFilePath = config('terraform-ls').get<string>('terraformLogFilePath', '');
-  const excludeModulePaths = config('terraform-ls').get<string[]>('excludeRootModules', []);
-  const ignoreDirectoryNames = config('terraform-ls').get<string[]>('ignoreDirectoryNames', []);
-
+  const terraformExecPath = config('terraform').get<string>('languageServer.terraform.executable.path', '');
+  const terraformExecTimeout = config('terraform').get<string>('languageServer.terraform.executable.timeout', '');
+  const terraformLogFilePath = config('terraform').get<string>('languageServer.terraform.executable.logFilePath', '');
+  const ignoreDirectoryNames = config('terraform').get<string[]>('languageServer.ignoreDirectoryNames', []);
   const ignoreSingleFileWarning = config('terraform').get<boolean>('languageServer.ignoreSingleFileWarning', false);
+  const experimentalFeatures = config('terraform').get('experimentalFeatures');
 
+  // deprecated
```

Comment:
I think we have not deprecated these options _yet_?

Reply:
They are listed as deprecated in package.json

Reply:
Oh right, just have not done that yet on the LS side then. 😅